bookmarklet/bookmarklet.js

```diff
@@ -173,12 +171,12 @@
 
 	// Load jQuery from CDN if needed
 	if (!window.jQuery) {
-		var head = document.getElementsByTagName("head")[0],
-				jQueryScript = document.createElement("script");
-		jQueryScript.src  = "http://code.jquery.com/jquery-1.10.0.min.js";
+		const head = document.getElementsByTagName("head")[0];
+		const jQueryScript = document.createElement("script");
+		jQueryScript.src = "http://code.jquery.com/jquery-1.10.0.min.js";
 		jQueryScript.onload = function() { startBookmarklet(window.jQuery); };
 		head.appendChild(jQueryScript);
 	} else {
 		startBookmarklet(window.jQuery);
 	}
-})();
+}());
```

Comment:
It's in our new ESLint rules, I think it's mostly a style preference. Crockford for instance doesn't like the other style because it looks like "dog balls"